x-pack/plugins/integration_assistant/server/templates/base-fields.yml.njk

```diff
@@ -0,0 +1,20 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset name.
+- name: data_stream.namespace
+  type: constant_keyword
+  description: Data stream namespace.
+- name: event.module
+  type: constant_keyword
+  description: Event module
+  value: {{ module }}
+- name: event.dataset
+  type: constant_keyword
+  description: Event dataset
+  value: {{ dataset }}
+- name: "@timestamp"
+  type: date
+  description: Event timestamp.
```

Comment:
Is this able to depend on the `external: ecs` definition to avoid duplicating definitions and types?

```suggestion
- name: data_stream.type
  external: ecs
- name: data_stream.dataset
  external: ecs
- name: data_stream.namespace
  external: ecs
- name: event.module
  type: constant_keyword
  external: ecs
  value: {{ module }}
- name: event.dataset
  type: constant_keyword
  external: ecs
  value: {{ dataset }}
- name: "@timestamp"
  external: ecs
```

Reply:
Unfortunately not, as its not triggering elastic-package.
However all ECS fields we dont need to define manually for some reason should really be removed from the templates as it will be constrained to 8.13+, which has the dynamic ecs templates.

There is an issue open on reviewing all the template files, but it has not been performed yet.